opendcs-rest-api/src/test/java/org/opendcs/odcsapi/sec/BaseIT.java

```diff
@@ -0,0 +1,84 @@
+/*
+ *  Copyright 2024 OpenDCS Consortium and its Contributors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License")
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.opendcs.odcsapi.sec;
+
+import javax.servlet.http.HttpServletResponse;
+
+import io.restassured.RestAssured;
+import io.restassured.filter.session.SessionFilter;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.opendcs.odcsapi.fixtures.TomcatServer;
+import org.slf4j.Logger;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+public abstract class BaseIT
```

Comment:
The tomcat and database operations would probably work better as an extension than just a base class.

Also possible I've just written way too many extensions and gotten used to the idea.

Reply:
good idea